api/holodeck/v1alpha1/types.go

```diff
@@ -189,7 +189,7 @@ type ExtraPortMapping struct {
 	HostPort      int `json:"hostPort"`
 }
 
-type NVContainerToolKit struct {
+type NVContainerToolkit struct {
```

Comment:
Just to clarify. These types are actually private?

Reply:
we could move this to `pkg/apis` as the controller-runtime recommends

Reply:
if we make the api public, we could potentially use holodeck all from go code, without a config file , it will be possible to import the `v1alpha1` spec and then call the create/delete functions.
maybe a good TODO for `v0.1.1`

Reply:
https://github.com/NVIDIA/holodeck/issues/15